awx/ui_next/src/components/CheckboxListItem/CheckboxListItem.jsx

```diff
@@ -44,7 +44,6 @@ const CheckboxListItem = ({
             <DataListCell key="name">
               <label
                 id={`check-action-item-${itemId}`}
-                htmlFor={`selected-${itemId}`}
```

Comment:
cc @tiagodread I think removing this would be fine for both a11y and test selectors but I wanted to double-check.

Reply:
@jakemcdermott from the e2e tests we're good, for a11y, I'm not _master_ (I'm still learning) but I think doing so isn't a good idea because:

> The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.

and 

> You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Another thing to consider is about these [Accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#accessibility_concerns)

> Don't place interactive elements such as anchors or buttons inside a label. Doing so makes it difficult for people to activate the form input associated with the label.

Running the axe tool against a modal for example was found 2 issues none of them related to `labels`:

![image](https://user-images.githubusercontent.com/12464858/108369588-826e2d80-71da-11eb-86d9-2deb47566dbd.png)

![image](https://user-images.githubusercontent.com/12464858/108369625-8b5eff00-71da-11eb-9f82-d2a2cb36f28f.png)